Small triage docs improvements (#721)
rust-lang/rust-forge: src/release/issue-triaging.md

      You can of course answer the question too :) (but make sure to mention that the user should go to URLO/Discord next time).
- Add appropriate labels ([Labels](#labels))
    - Specifically, `T-*` and `C-*` are the most relevant
- If the issue contains no reproduction but needs one (when in doubt, it needs one), ask for one and add the `S-needs-repro` label

> - The issue tracker is the wrong place for some kinds of feature requests. Tell the author about it.

Reviewer: ```suggestion
- The issue tracker is the wrong place for some kinds of feature requests. Suggest the author where they can get support.
```
